aydin/nn/models/torch/torch_unet.py

```diff
@@ -146,20 +139,109 @@ def _decoder_convolutions(self):
         return convolutions
 
 
-def n2t_unet_train_loop(
+def n2s_train(
```

Comment:
Is this function particular to unet model or can it accept other models?

Reply:
it can accept other models